javascript-if/main.js

```diff
@@ -0,0 +1,81 @@
+/* exported isUnderFive,
+            isEven,
+            startsWithJ,
+            isOldEnoughToDrink,
+            isOldEnoughToDrive,
+            isOldEnoughToDrinkAndDrive,
+            categorizeAcidity,
+            introduceWarnerBro
+ */
+function isUnderFive(number) {
+  if (number < 5) {
+    return true;
+  } else {
+    return false;
+  }
+
+}
+
+function isEven(number) {
+  if (number % 2 > 0) {
+    return false;
+  } else {
+    return true;
+  }
+}
+
+function startsWithJ(string) {
+  if (string.charAt(0) === 'J') {
+    return true;
+  } else {
+    return false;
+  }
+}
+
+function isOldEnoughToDrink(person) {
+  if (person.age < 21) {
+    return false;
+  } else {
+    return true;
+  }
+}
+
+function isOldEnoughToDrive(person) {
+  if (person.age < 16) {
+    return false;
+  } else {
+    return true;
+  }
+}
+
+function isOldEnoughToDrinkAndDrive(person) {
+  if ((person.age < 18) && (person.age < 21)) {
+    return false;
+  } else {
+    return true;
+  }
+}
```

Comment:
This should always return `false`:

---

![image](https://user-images.githubusercontent.com/16143771/145285629-d949b911-4ca2-4df6-8c35-ef2c3fb2e9bb.png)

Reply:
oh I just realized!!!!